src/screens/NewTopicScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { postNewTopic}from '../api/DataHandler';
+
+//import { Link } from 'react-router-dom'
+
+interface NewTopicState{
+    post_title: string,
+    post_text: string,
+    post_url: string,
+    username: string,
+    pwdHash: string,
+    hanesst_id: number,
+}
+class NewTopicScreen extends React.Component<any, NewTopicState> {
+
+state = {
+    post_title: "",
+    post_text: "",
+    post_url: "",
+    username: "",
+    pwdHash: "",
+    hanesst_id: 0
+}
+
+    
+handlePostTitleInput(event:React.FormEvent<HTMLInputElement>){
+    this.setState({post_title: event.currentTarget.value})
+}
```

Comment:
**Det der er virkelig godt. +1**

Reply:
Only took me a couple of hours to figure out how to do...